chsdi/templates/htmlpopup/reisezeit_agglomerationen_miv.mako

```diff
@@ -0,0 +1,7 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('ch.are.reisezeit-agglomerationen-miv.verkehrszone_id')}</td><td>${c['attributes']['verkehrszone_id'] or '-'}</td></tr>
```

Comment:
@Simon1193 
the attribute ``verkehrszone_id`` is already mapped as primary key in the model. 
You have to use ``c['featureId']`` in the mako to access this attribute or the primary key from the model.

Reply:
Thanks a lot for your answer. I will adapt the mako